src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -158,44 +105,6 @@ define(function (require, exports, module) {
         },
 
         /**
-         * Scrolls the layers panel to make (newly) selected layers visible.
-         *
-         * @param {LayerStructure} layerStructure
-         */
-        _scrollToSelection: function (layerStructure) {
```

Comment:
I'm glad this logic was simplified. Do you know of any changes in behavior in the new implementation?